contrib/charts/cert-manager/README.md

```diff
@@ -54,8 +54,8 @@ The following table lists the configurable parameters of the cert-manager chart
 | Parameter | Description | Default |
 | --------- | ----------- | ------- |
 | `image.repository` | Image repository | `quay.io/jetstack/cert-manager-controller` |
-| `image.tag` | Image tag | `v0.3.0` |
-| `image.pullPolicy` | Image pull policy | `IfNotPresent` |
+| `image.tag` | Image tag | `canary` |
+| `image.pullPolicy` | Image pull policy | `Always` |
```

Comment:
Is it safe to Always pull the canary image?
What if a Pod restarts and downloads a newer version of the canary image than the other pods are running?
Wouldn't it be better to use `IfNotPresent`?

Reply:
No - using IfNotPresent might *seem* to allow you to stick to a single revision, but in reality, as the pod gets scheduled onto different nodes around the cluster, you might find yourself running 5 different versions.

Kubernetes specifically sets `pullPolicy` to `Always` if the `latest` image is being used, so we are doing the same 😄